pkg/v1/providers/infrastructure-aws/v1.2.0/cconly/base.yaml

```diff
@@ -279,6 +293,57 @@ spec:
             {{- end }}
             {{- end }}
 
+  - name: AWSCT_CNIIngressRules_Calico
+    enabledIf: '{{if (eq .cni "calico") }}true{{end}}'
+    definitions:
+    - selector:
+        apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+        kind: AWSClusterTemplate
+        matchResources:
+          infrastructureCluster: true
+      jsonPatches:
+      - op: add
+        path: "/spec/template/spec/network/cni/cniIngressRules/-"
+        valueFrom:
+          template: |
+            description: bgp (calico)
+            fromPort: 179
+            protocol: tcp
+            toPort: 179
+      - op: add
+        path: "/spec/template/spec/network/cni/cniIngressRules/-"
+        valueFrom:
+          template: |
```

Comment:
This doesn't actually have a templated value right? couldn't this be
```
value: |
  description: IP-in-IP (calico)
  ...
```

Reply:
Looks like setting this to value doesn't work.
```
  - spec.patches[1].definitions[0].jsonPatches[0].valueFrom: Invalid value: "{}": valueFrom must set either template or variable
```

Reply:
Needed to do remove `|` when using `value` and not use `valueFrom`. Updated